sdk/monitor/azquery/assets.json

```diff
@@ -2,5 +2,5 @@
   "AssetsRepo": "Azure/azure-sdk-assets",
   "AssetsRepoPrefixPath": "go",
   "TagPrefix": "go/monitor/azquery",
-  "Tag": "go/monitor/azquery_98fd781041"
+  "Tag": "go/monitor/azquery_123118d585"
```

Comment:
Why does this need to change}? (I don't know how it's supposed to work 🤭 feel free to point me at docs)

Reply:
I had to rerecord the test for QueryResource. Aggregation went from "Average,count" to "Average,Count" in the request since I started using the built in types in the example.

Reply:
This is the equivalent to "I've updated the recordings".